src/metabase/api/permissions.clj

```diff
@@ -153,8 +153,8 @@
   "Fetch the details for a certain permissions group."
   [id]
   (validation/check-group-manager id)
-  (-> (db/select-one PermissionsGroup :id id)
-      (hydrate :members)))
+  (api/check-404 (-> (db/select-one PermissionsGroup :id id)
+      (hydrate :members))))
```

Comment:
The indentation is a bit wonky here. I'd recommend reading through the Clojure style guide [here](https://guide.clojure.style) since we usually try to adhere to it.

The section that applies here is [Function Arguments Indentation](https://guide.clojure.style/#one-space-indent). You can either keep the arg to `check-404` on the same line like you currently have it, but then the two arguments to the `->` should be aligned to the same level. Or you can put the entire `->` form on a new line. I'd probably do the latter here (see the change I suggested) but it's up to you.
```suggestion
  (api/check-404 
   (-> (db/select-one PermissionsGroup :id id)
       (hydrate :members))))
```

Reply:
Is it ok if I format other parts of the file? I saw some code that doesn't adhere to the property when I turned on Format On Save on VSCode, i.e.:

https://github.com/metabase/metabase/blob/952f7523c4ccc8cf9100131f2728c64b05eb611f/src/metabase/api/permissions.clj#L165-L166

Reply:
Yeah that's fine. I think that formatting for DB calls is pretty common but I don't think we have a good reason for it. That one can probably be one line anyway.